blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -125,6 +125,10 @@ private static void startIngestServices(ScheduleContext context) {
                     System.exit(1);
                 }
             }
+
+            final Thread eerThread = new Thread(ExcessEnumReader.getInstance(), "Excess Enum Table Reader");
+            eerThread.start();
```

Comment:
Let's add another log.info here that the ExcessEnumRead thread was started so we can note it in the log like when other ingestion module was started in the loop before it.  We also might or might not increment services_started counter by 1, depending if you think if it should be consider part of the "ingestion services" started by this method.